types/node/timers.d.ts

```diff
@@ -77,15 +77,15 @@ declare module 'timers' {
         namespace setInterval {
             const __promisify__: typeof setIntervalPromise;
         }
-        function clearInterval(intervalId: NodeJS.Timeout | undefined): void;
```

Comment:
I'm not sure why these previously had `| undefined` so I just left that as-is.

Reply:
For parity with their counterparts in browsers i.e. parity with lib.dom.d.ts